Disambiguate "it"

andynava opened the pull request:
Took me a while to figure out what "it" was referring to. I believe it is "this article".

azurecla commented:
Hi __@adhurwit__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

BryanLa commented:
approving per conversation w/Carol Bailey

andynava committed:
Disambiguate "it"

Took me a while to figure out what "it" was referring to. I believe it is "this article".

andynava commented on articles/monitoring-and-diagnostics/monitoring-overview.md:
```diff
@@ -24,36 +24,36 @@ This article provides an overview of the tools and services involved in holistic
 All Azure monitoring services are now available in a single UI pane. For more information on how to access this area, see [Get started with Azure Monitor](monitoring-get-started.md). 
 
-You can also access monitoring functions for specific resources by highlighting those resources and drilling down into their monitoring options. 
+You can also access monitoring functions for specific Azure resources by highlighting those resources and drilling down into their monitoring options. 
```
I believe this is only available for Azure resources

andynava commented on articles/monitoring-and-diagnostics/monitoring-overview.md:
```diff
@@ -24,36 +24,36 @@ This article provides an overview of the tools and services involved in holistic
 **The best option is to use Application Insights, Azure Monitor, and Visual Studio together**
 
-Azure now provides the full power of the Visual Studio debugger in the cloud. Configure Azure Monitor to send telemetry to Application Insights. Enable Visual Studio to include the Application Insights SDK in your application. Once in Application Insights, you can use the Application Map to discover visually which parts of your running application are unhealthy or not. For those parts that are no ...
+Azure now provides the full power of the Visual Studio debugger in the cloud. Configure Azure Monitor to send telemetry to Application Insights. Enable Visual Studio to include the Application Insights SDK in your application. Once in Application Insights, you can use the Application Map to discover visually which parts of your running application are healthy or not. For those parts that are not  ...
```
"unhealthy or not" feels like negating a negative. Changed to "healthy or not"

andynava commented on articles/monitoring-and-diagnostics/monitoring-overview.md:
```diff
@@ -64,19 +64,20 @@ For more information, see [Monitoring Web Apps](../application-insights/app-insi
 **The best option is to use Application Insights and if possible Visual Studio for the full debugging experience.**
 
-Use the Application Insights Snapshot Debugger to debug your app. When a certain error threshold occurs with production components, the system automatically captures telemetry in windows of time called “snapshots." The amount captured is safe for a production cloud because it’s small enough not to affect performance but significant enough to allow tracing.  The system can capture multiple snapsho ...
+Use the Application Insights Snapshot Debugger to debug your app. When a certain error threshold occurs with production components, the system automatically captures telemetry in windows of time called “snapshots." The amount captured is safe for a production cloud because it’s small enough not to affect performance but significant enough to allow tracing.  The system can capture multiple snapsho ...
```
RBAC roles are a very different concept than monitoring, so added link to article describing them

andynava commented on articles/monitoring-and-diagnostics/monitoring-overview.md:
```diff
@@ -64,19 +64,20 @@ For more information, see [Monitoring Web Apps](../application-insights/app-insi
 
 **Same as scenario 1. Use Application Insights, Azure Monitor, and Visual Studio together**
+
 Application Insights also supports gathering telemetry from processes running inside containers and from microservices (Kubernetes, Docker, Azure Service Fabric). For more information, [see this video on debugging containers and microservices](https://go.microsoft.com/fwlink/?linkid=848184). 
```
Separated subheading from the paragraph description

andynava commented on articles/monitoring-and-diagnostics/monitoring-overview.md:
```diff
@@ -64,19 +64,20 @@ For more information, see [Monitoring Web Apps](../application-insights/app-insi
 ### Scenario 4 – Fix performance issues in your Azure application
 
-The [Application Insights profiler](../application-insights/app-insights-profiler.md) is designed to help troubleshoot these types of issues. You can identify and troubleshoot performance issues for applications running in App Services (Web Apps, Logic Apps, Mobile Apps, API Apps) and other compute resources such as Virtual Machines, Virtual machine scale sets (VMSS), Cloud Services, and Service  ...
+The [Application Insights profiler](../application-insights/app-insights-profiler.md) is designed to help troubleshoot these types of issues. You can identify and troubleshoot performance issues for applications running in App Services (Web Apps, Logic Apps, Mobile Apps, API Apps, Function Apps) and other compute resources such as Virtual Machines, Virtual machine scale sets (VMSS), Cloud Service ...
```
Azure Function Apps also run in an App Service
